client/rpc/block.go

```diff
@@ -133,6 +133,7 @@ func BlockRequestHandlerFn(cliCtx context.CLIContext) http.HandlerFunc {
 			w.Write([]byte(err.Error()))
 			return
 		}
+		w.Header().Set("Content-Type", "application/json")
```

Comment:
@HaoyangLiu can you move all the JSON headers to the first lines of the corresponding functions ? otherwise the first return cases won't have the header

Reply:
I added it at the end of handler, because if an error is return, the response is not JSON pattern.

Reply:
What do you think about my concerns?

Reply:
ok, no problem